core/crypto/crypto.c

```diff
@@ -0,0 +1,440 @@
+/*
+ * Copyright (c) 2017, Linaro Limited
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <compiler.h>
+#include <kernel/panic.h>
+#include <tee/tee_cryp_provider.h>
+
+#if !defined(_CFG_CRYPTO_WITH_HASH)
+TEE_Result crypto_hash_get_ctx_size(uint32_t algo __unused,
+				    size_t *size __unused)
+{
+	return TEE_ERROR_NOT_SUPPORTED;
```

Comment:
`TEE_ERROR_NOT_IMPLEMENTED` instead, for consistency ?
Only hash function use here `TEE_ERROR_NOT_SUPPORTED`. Others uses `TEE_ERROR_NOT_IMPLEMENTED`.

Reply:
Thanks, I'll replace all those.